### PR TITLE
Fix common typos

### PR DESCRIPTION
**Pull request by jtescher**
Fixing some common typos. Unsure about typos in method names, I can remove those fixes if you would like to handle them in another PR or some other way.

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
LGTM (as far as I can see all the changed methods are private internal ones so isn't a problem with BC)

**Comment by ktoso** [comment]
Checked the methods as well, LGTM - thanks @jtescher !

**Comment by jtescher** [comment]
Awesome!

File: akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala [hunk 1]
```diff
@@ -560,7 +560,7 @@ private[stream] object Fusing {
      * connections within imported (and not dissolved) GraphModules.
      * See also the comment in addModule where this is partially undone.
      */
-    def registerInteral(s: Shape, indent: Int): Unit = {
+    def registerInternal(s: Shape, indent: Int): Unit = {
       if (Debug) println("  " * indent + s"registerInternals(${s.outlets.map(hash)})")
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala]: Oh wait, by looking at the debug println I think this method should actually be called `registerInternals` - could you fix that up in this PR @jtescher please?

Comment (jtescher) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala]: Ah sure thing :+1:

**Comment by jtescher** [comment]
@ktoso looking better? Want to re-run tests?

File: akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala [hunk 2]
```diff
@@ -560,7 +560,7 @@ private[stream] object Fusing {
      * connections within imported (and not dissolved) GraphModules.
      * See also the comment in addModule where this is partially undone.
      */
-    def registerInteral(s: Shape, indent: Int): Unit = {
+    def registerInternals(s: Shape, indent: Int): Unit = {
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala]: better, thanks!

**Comment by ktoso** [comment]
Better safe than sorry so let's wait for validation to complete and merge then :)
Thanks again!

**Comment by ktoso** [comment]
Seems we're good here, thanks :)

**Pull request merged**